eng/performance/maui_scenarios_ios.proj

```diff
@@ -11,8 +11,13 @@
     <PreparePayloadOutDirectoryName>scenarios_out</PreparePayloadOutDirectoryName>
     <PreparePayloadWorkItemBaseDirectory Condition="'$(TargetsWindows)' == 'true'">$(CorrelationPayloadDirectory)$(PreparePayloadOutDirectoryName)\</PreparePayloadWorkItemBaseDirectory>
     <PreparePayloadWorkItemBaseDirectory Condition="'$(TargetsWindows)' != 'true'">$(CorrelationPayloadDirectory)$(PreparePayloadOutDirectoryName)/</PreparePayloadWorkItemBaseDirectory>
+    
     <NativeAOTCommandProps Condition="'$(RuntimeFlavor)' == 'coreclr'">--nativeaot true</NativeAOTCommandProps>
-    <HybridGlobalizationCommandProps Condition="'$(HybridGlobalization)' == 'true'">--hybrid-globalization true</HybridGlobalizationCommandProps>
+    <!-- For non-default configurations, add the configuration to the name of the test (in the matching format), otherwise didn't add anything.
```

Comment:
Thanks, LGTM!

I suggest we disable the startup measurements to get it merged this week, and try to resolve https://github.com/dotnet/performance/issues/3148. Additionally, I've opened https://github.com/dotnet/performance/issues/3318 to ensure unique identifiers for measurements in general.